rails/config/locales/en/dashboard.en.yml

```diff
@@ -12,6 +12,14 @@ en:
     online: Online Map Configuration
     settings: Map Settings
     settings_description: Customize your community's map settings.
+    projection:
+      mercator: Mercator
+      albers: Albers
+      equalEarth: Equal Earth
+      equirectangular: Equirectangular
+      lambertConformalConic: Lambert Comformal Conic
+      naturalEarth: Natural Earth
+      winkelTripel: Winkel Tripel
```

Comment:
Let's add these to all of the other translation files, too, other we'll get translation missing texts.